docs/kafka/access-mgmt-kafka/README.adoc

```diff
@@ -475,6 +477,37 @@ h|Operations
 |===
 --
 
+[id="proc-user-account-access_{context}"]
+== Granting users access to view other user accounts
+
+[role="_abstract"]
+As an organization administrator, you can use Role-Based Access Control (RBAC) to allow users to view other users in a group.
+
+You set up access by assigning a predefined role called `User Access principal viewer` to a user group.
+By assigning the role, users within the group are able to do the following:
+
+* View and select other users when changing owners and managing access to Kafka instances in the web console
+* Specify user names when updating the owner of a Kafka instance using the `rhoas kafka update` command of the `rhoas` CLI for {product-long-kafka}
+
+.Prerequisites
+* You're logged into the {org-name} web console as an organization administrator.
+* A user group contains the users to assign the role to.
+
+NOTE: If you want to add the `User Access principal viewer` role to a single user, create a new group for that user only.
+
+ifndef::community[]
+For more information on setting up user access in the web console, see the link:https://access.redhat.com/documentation/en-us/red_hat_hybrid_cloud_console/2021[_User Access Configuration Guide for Role-based Access Control (RBAC)_^].
+endif::[]
+
+.Procedure
+
+. From the home page of the web console, click image:img-configuration-gear.png[] (*Settings*).
```

Comment:
@sterobin -- not sure we want to introduce this icon image. It's used in the doc I link to.

Reply:
@PaulRMellor No, I wouldn't use the icon. What we've been doing so far for unlabeled icons is to just say the label and parenthetically describe the icon. Sounds funny, but is optimal for accessibility. Example, throughout the docs we say "select the options icon (three vertical dots)". So likewise, I suggest "click Settings (gear icon)".

That said, I noticed that the gear icon actually has a couple of options right now: "Settings" and either "Use the beta release" or "Stop using the beta release" depending on which you're in. I don't know how much longer that will be there so maybe okay just to consider the gear Settings, and even if not, intuitive enough to figure out? Wdyt?

Either way, I recommend the following streamlining of the first couple of steps to be consistent with the language and approach currently used in this and other docs:

```
. In the upper-right corner of the {product-kafka} web console, go to *Settings* (gear icon) > *User Access* > *Groups* and click the name of the user.
. ....
```

If you think that's too long and that breaking it up with Settings and then the others separately is best, that's fine, but we probably still want to stick with "In the upper-right corner of the {product-kafka} web console". And I know that it probably needs to be a new attribute for the web console as "Application Services" in community and "Red Hat Hybrid Cloud Console" for enterprise, but that's outside the scope of this PR and best to just be consistent with the term already used throughout the doc.

Reply:
Thanks, @sterobin. I dropped the icon. Consolidating the the first steps is fine too.